src/app/store/index.ts

```diff
@@ -86,12 +87,12 @@ export const store = configureStore({
   enhancers:
     process.env.WALLET_ENVIRONMENT === 'development'
       ? [
-          devToolsEnhancer({
-            hostname: 'localhost',
-            port: 8000,
-            realtime: true,
-            suppressConnectErrors: false,
-          }),
+          // devToolsEnhancer({
```

Comment:
@fbwoolf disabling this gets the unit tests to pass as the issue lies in `@redux-devtools/remote` using a new version of `nanoid`